script/integration-test

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+
+# subshell so that we can export PATH without breaking other things
+(
+	source "$(dirname "$BASH_SOURCE")/.integration-daemon-start")
```

Comment:
If this only runs integration tests (without starting the daemon), I think we could move it to `test/integration/run.sh` - not sure about that though.

Thoughts?